04-gas/gas-section-one/gas-section-one.html

```diff
@@ -16,7 +16,9 @@
 
       <div class="gas-section-one-copy-container">
         
-        <h1 class="gas-section-one-title">1. Lower your <span class="bold">water temperature</class></h1>
+        <div class="gas-section-one-title-container">
```

Comment:
add this class to the top of the css for temporary border

Reply:
done